src/providers/SubscriptionProvider/SubscriptionProvider.constants.js

```diff
@@ -1,4 +1,15 @@
-export const UPDATE_PRODUCT_STATE = 'UPDATE_PRODUCT_STATE';
+export const UPDATE_ANDROID_SUBSCRIPTION_STATE =
+  'UPDATE_ANDROID_SUBSCRIPTION_STATE';
 export const UPDATE_SUBSCRIBER_ID = 'UPDATE_SUBSCRIBER_ID';
 export const UPDATE_IS_SUBSCRIBED = 'UPDATE_IS_SUBSCRIBED';
 export const UPDATE_SUBSCRIPTION = 'UPDATE_SUBSCRIPTION';
+export const UPDATE_SUBSCRIPTION_ERROR = 'UPDATE_SUBSCRIPTION_ERROR';
```

Comment:
Constants for Reducers should have the format `cboard/subscription/'UPDATE_SUBSCRIPTION_ERROR`